Editor/AGS.Types/Character.cs

```diff
@@ -45,7 +45,7 @@ public class Character : ICustomTypeDescriptor, IToXml, IComparable<Character>
         private bool _adjustSpeedWithScaling;
         private bool _adjustVolumeWithScaling;
         private bool _movementLinkedToAnimation = true;
-        private CustomProperties _properties;
+        private CustomProperties _properties = new CustomProperties(CustomPropertyAppliesTo.Characters);

```

Comment:
This fix does not work, because _properties member is re-created inside SerializeUtils.DeserializeFromXML call, and its "applies to" gets reset to the Character once again.

Reply:
If I understood, the Xml constructor sets the Properties property that has the CustomProperties, so I just added an additional set after to force the appliesTo to be correct in https://github.com/adventuregamestudio/ags/pull/2604/commits/c1ce7723c7f940a5984ed4f5c18dc3a377486094